setup.py

```diff
@@ -149,7 +184,14 @@ def try_run_setup(**kwargs):
             try_run_setup(**kwargs)
         elif kwargs["with_binary"]:
             kwargs["with_binary"] = False
-            print("WARNING: The C extension could not be compiled, sklearn tree models not supported.")
+            print(
```

Comment:
this is fine. But just FYI I increased the pylint line limit just to avoid wraps like this.